lib/ansible/module_utils/k8s/raw.py

```diff
@@ -271,6 +279,33 @@ def perform_action(self, resource, definition):
                             self.fail_json(msg="Resource deletion timed out", **result)
                 return result
         else:
+            if self.apply:
+                if self.check_mode:
+                    k8s_obj = definition
+                else:
+                    try:
+                        k8s_obj = resource.apply(definition, namespace=namespace).to_dict()
+                    except DynamicApiError as exc:
+                        msg = "Failed to replace object: {0}".format(exc.body)
```

Comment:
I would use `apply` rather than `replace` in the error message, since `replace` already has a specific meaning.

Reply:
Fixed, thanks